src/ts/player-events.d.ts

```diff
@@ -473,11 +473,14 @@ declare namespace bitmovin {
       html?: string;
       region?: string;
       regionStyle?: string;
-      position?: string;
```

Comment:
The position property in the player is also defined as `string`. Is that wrong? Do we need to adjust that in the player API? Or can it be a string too in certain cases?